src/test/test_hs_service.c

```diff
@@ -2172,6 +2172,50 @@ test_export_client_circuit_id(void *arg)
   tor_free(cp2);
 }
 
+static void
+test_circ_timeout(void *arg)
```

Comment:
Hmm the unittest failed on clang.

Reply:
I believe I've fixed that with fixup: 28ce878609730cc143a8ea3ef0f012f34cdbc0f4